build-logic-commons/settings.gradle.kts

```diff
@@ -21,7 +21,7 @@ dependencyResolutionManagement {
 }
 
 plugins {
-    id("org.gradle.toolchains.foojay-resolver-convention") version("0.8.0")
+    id("org.gradle.toolchains.foojay-resolver-convention") version("0.9.0")
```

Comment:
🤔  This is a strange syntax: it's an infix call without a dot, yet the argument is in parenthesis